core/src/components/setup/RecommendedApps.vue

```diff
@@ -79,7 +79,7 @@ const recommended = {
 		description: t('core', 'Simple email app nicely integrated with Files, Contacts and Calendar.'),
 		icon: imagePath('core', 'actions/mail.svg'),
 	},
-	talk: {
+	spreed: {
 		description: t('core', 'Screensharing, online meetings and web conferencing – on desktop and with mobile apps.'),
```

Comment:
While not entirely wrong, the description suggestions there is a desktop client 😅

Also I'd like to outline the more "private" parts of Talk. It's not only about the big usecases, but also your simple 1-1 chats and stuff
```suggestion
		description: t('core', 'Chatting, video calls, screensharing, online meetings and web conferencing – in your browser and with mobile apps.'),
```

Reply:
fine by me :wink: